doc/how-to-run-in-windows.rst

```diff
@@ -0,0 +1,143 @@
+How to use Cabal in Windows
+===========================
+
+This document describes how to use Cabal in a Windows system. See the
+:ref:`Further reading` section for some other references that might provide some
+more explanations. For a TL;DR, jump to the :ref:`Complete configuration`.
+
+Install the Haskell environment
+-------------------------------
+
+The recommended way of setting up a Haskell environment in Windows is by using
+`GHCup <https://www.haskell.org/ghcup/>`_. Follow the steps outlined in its
+webpage to install at least GHC and Cabal. GHCup will install its own MSYS2
+system in your computer unless told not to do so, refer to `its documentation
+<https://www.haskell.org/ghcup/install/#windows_1>`_ for more information.
+
+Ensure that Cabal can call the tools it needs
+---------------------------------------------
+
+Cabal sometimes needs to call tools that do not come with Windows (such as
+``make`` or even ``git``). The MSYS2 project makes many of them available on
+Windows. The directories where those are located need to be made visible in the
+``PATH`` when executing ``cabal``. For that, Cabal provides the
+``extra-prog-path`` configuration option. Your :ref:`global configuration
+<config-file-discovery>` should include this option:
+
+::
+
+   extra-prog-path: <msys-dir>\<environment>\bin
+                    <msys-dir>\usr\bin
+
+Where ``<msys-dir>`` points to the location of your MSYS2 installation. Refer to
+GHCup's documentation on where this directory is located by default.
+``<environment>`` has to be one of the environments of MSYS2, which for GHCup is
+``mingw64``. You can learn more about the different environments in the `MSYS2
+documentation <https://www.msys2.org/docs/environments/>`_.
+
+.. note::
+
+   Currently this step is already done by the GHCup installation on your behalf
+   by default unless told otherwise.

```

Comment:
This feels a little redundant to me. I’d suggest instead: ‘Unless told otherwise, GHCup already adds these directories to `extra-prog-path` by default.’

Reply:
Done